chaos_kitten/toys_cli.py

```
@@ -0,0 +1,246 @@
+            with open(temp_path, "w", encoding="utf-8") as f:
+                f.write(content)
+
+            try:
+                validator = AttackProfileValidator()
+                report = validator.validate_profile(str(temp_path))
+                if not report.is_valid:
+                    console.print("[red]Schema validation failed:[/red]")
+                    for err in report.errors:
+                        console.print(f"  - {err}")
+                    temp_path.unlink()
+                    raise Exit(code=1)
```

Ruff (0.15.1)

[warning] Abstract `raise` to an inner function

(TRY301)